src/model_signing/_cli.py

```diff
@@ -458,12 +458,21 @@ def _verify_private_key(
 @_ignore_paths_option
 @_ignore_git_paths_option
 @_certificate_root_of_trust_option
+@click.option(
+    "--fingerprints",
```

Comment:
Can we change this, and the variable and keyword argument to be `show_fingerprints` or `log_fingerprints`? Makes it more informative

Reply:
Sure, I can change it.